### ISSUE TITLE
log.error shows empty object

### ISSUE THREAD

**Issue by fuggerstadt1907** [onubo/react-native-logs#25]
Hi all 👋 

I'm facing problems when logging errors in try catch blocks.
Caught and logged errors shows only an empty object 😞 

`19/01/2021, 08:23:55 | ERROR | {}`

Any ideas?` console.error(err)` does it right:
`Error: BLE_CONNECTION_ERROR: Cannot convert undefined or null to object`

**Comment by fuggerstadt1907** [comment]
Hi @alessandro-bottamedi sure, here is an example

```js
return (
    <CustomView
      style={styles.view}
      accessibilityLabel={A11y.DEVICE_INFO}
      fullSize={!!language}
      header={
        <AppHeader
          onPressClose={navigateToDetail}
          title={I18n.Interface.DEVICE_DETAILS.OPERATING_INSTRUCTIONS.HEADER_TITLE}
        />
      }>
      {!language &&
        OPERATING_INSTRUCTIONS_LANGUAGES.map(({ key, label }) => (
          <ListElement
            key={key}
            accessibilityLabel={label}
            title={label}
            bottomDivider
            chevron
            onPress={() => setLanguage(key)}
          />
        ))}
      {language && (
        <Pdf
          source={source}
          onError={error => log.error('Error while presenting PDF', error)}
          style={styles.pdf}
          scale={0.95}
          minScale={0.95}
        />
      )}
    </CustomView>
  )
```

`log.error('Error while presenting PDF', error)` produces the following log in my console:
```
20/01/2021, 08:07:52 | ERROR | Error while presenting PDF
20/01/2021, 08:07:52 | ERROR | {}
```
Also irritating, that this is printed in two separate lines 😞 

When I chance log.error to console.error the full error is printed properly


The logger is configured as follows
```js
import { logger } from 'react-native-logs'
import { colorConsoleSync } from 'react-native-logs/dist/transports/colorConsoleSync'

import { Config } from '../constants'

const defaultConfig = {
  severity: Config.LOGGER_SEVERITY,
  transport: colorConsoleSync,
  transportOptions: null,
  levels: {
    trace: 0,
    debug: 1,
    info: 2,
    warn: 3,
    error: 4,
  },
}
const log = logger.createLogger(defaultConfig)

export { log }
```

**Comment by alessandro-bottamedi** [comment]
I will soon release version 3 which will fix this bug and also concantenation on the same line, for now try using: 
```js
log.error('Error while presenting PDF', error.message)
```

**Comment by alessandro-bottamedi** [comment]
Fixed in v 3.0.0

**Comment by fuggerstadt1907** [comment]
thx @alessandro-bottamedi  I'll check out v3 today ✌️
